device/globals/global_vm.gd

```diff
@@ -425,6 +425,9 @@ func change_scene(params, context):
 	# It might be tempting to use `get_tree().change_scene(params[0])`,
 	# but this custom solution is safer around your scene structure
 	printt("change scene to ", params[0])
+	var start_pos_node = null
+	if params.size() == 2:
+		start_pos_node = params[1]
```

Comment:
Does `start_pos_node` need to be declared? I feel like a comment over the call to `main.set_scene()` explaining what `params[1]` might be would be better.

Reply:
I actually like your approach better now, it makes the prints also clearer. I'll force-push that.